doc/pages/administrator/060-setup_cluster.md

```diff
@@ -98,5 +78,13 @@ spec:
         memory: 6144M
 ```
 
-You can simply apply this resource in your Kubernetes cluster with `kubectl apply -f`. The Operator will take
-over from there.
+Starting from Astarte v1.0.1, traffic coming to the broker is TLS terminated ad VerneMQ level. The
+two fields controlling this features, namely `sslListener` and `sslListenerCertSecretName` can be
+found within the `vernemq` section of the Astarte CR. To provide more context on this topic:
```

Comment:
I'd rephrase this:
"..In a nutshell, their meaning is:
- `sslListener` controls whether TLS termination is enabled at VerneMQ level or not, "

Reply:
thanks